.travis.yml

```diff
@@ -6,11 +6,11 @@
   matrix:
     include:
       - os: linux
-        go: 1.7
+        go: 1.7.x
```

Comment:
yeah then we can test multiple versions like 1.7.x and 1.8.x just to see if there are incompatible changes, so I like as is